scripts/jenkins/dist-feed/dist-feed-manifest.yml

```diff
@@ -121,10 +121,10 @@ packages:
     export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/21.8') | latest_export }}"
     ipk_path: 'targets/linuxU/x64/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
   dist-nilrt-grub_22.0_arm:
```

Comment:
Update this text too? And the one below.

Reply:
Done, and updated commit/PR description.